kucher/view/main_window/telega_control_widget/task_specific_status_widget/fault_status_widget.py

```diff
@@ -83,12 +89,38 @@ def on_general_status_update(self, timestamp: float, s: GeneralStatusView):
         self._error_code_hex.setText(f'0x{tssr.failed_task_exit_code:02X}')
         self._error_code_bin.setText(f'0b{tssr.failed_task_exit_code:08b}')
 
-        self._error_description_display.setText('(elaboration not available)')
+        file_name = get_absolute_path('view', 'main_window', 'telega_control_widget', 'task_specific_status_widget',
+                                      f'error_codes.yml', check_existence=True)
+        with open(file_name, 'r') as f:
+            error_codes = yaml.safe_load(f)
+
+        failed_task_name = str(tssr.failed_task_id).split('.')[-1]
+
+        error = error_codes[failed_task_name].get(tssr.failed_task_exit_code, 'unknown error')
+        error_description = error.get('description', 'unknown error') if type(error) is dict else error
+        error_comment = error.get('comment', '') if type(error) is dict else ''
 
-    def _make_display(self, tool_tip: str = ''):
+        self._error_description_display.setText(error_description)
+        self._error_comment_display.setText(error_comment)
+
+    def _make_line_display(self, tool_tip: str = ''):
         o = QLineEdit(self)
         o.setReadOnly(True)
         o.setFont(get_monospace_font())
         o.setAlignment(Qt.AlignCenter)
         o.setToolTip(tool_tip)
         return o
+
+    def _make_text_display(self, tool_tip: str = ''):
+        o = QTextEdit(self)
+        o.setReadOnly(True)
+        o.setLineWrapMode(True)
+
+        font = get_monospace_font()
+        font.setItalic(True)
+        o.setFont(font)
+
+        o.setStyleSheet("border: 0px;")
```

Comment:
I would prefer to have it look like a regular bordered text box, like the line display above it but multi-line. Currently, the text is rendered like a regular label, so it's easy to mistake it for part of the GUI structure as opposed to an informational message.